.github/scripts/remote-execute-integration-tests.sh

```diff
@@ -30,30 +30,9 @@ function upload_report_to_buildkite() {
 
   # Remove all non-valid XML characters. We set -CSDA to make Perl use UTF8 for
   # its I/O and -p to make it loop over the whole file.
-  perl -CSDA -pe 's/[^\P{C}\t\n\r]//g' ./report.xml > ./report-stripped.xml
-
-  local -r report_path="$PWD/report-stripped.xml"
-
-  curl \
-    -X POST \
-    --fail-with-body \
-    -H "Authorization: Token token=\"$BUILDKITE_ANALYTICS_TOKEN\"" \
-    -F "data=@$report_path" \
-    -F "format=junit" \
-    -F "run_env[CI]=github_actions" \
-    -F "run_env[key]=$GITHUB_ACTION-$GITHUB_RUN_NUMBER-$GITHUB_RUN_ATTEMPT" \
-    -F "run_env[number]=$GITHUB_RUN_NUMBER" \
-    -F "run_env[branch]=$GITHUB_REF" \
-    -F "run_env[commit_sha]=$GITHUB_SHA" \
-    -F "run_env[message]=$git_commit_message" \
-    -F "run_env[url]=https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID" \
-    -F "tags[architecture]=$MATRIX_SYSTEM" \
-    -F "tags[nix_flakeref]=github:flox/flox/$GITHUB_SHA" \
-    -F "tags[nix_attribute]=packages.$MATRIX_SYSTEM.flox-cli-tests" \
-    https://analytics-api.buildkite.com/v1/uploads \
-    || true
+  perl -CSDA -pe 's/[^\P{C}\t\n\r]//g' ./report.xml > ./report-clean.xml
```

Comment:
is this still necessary with codecov?
Buildkite had some parsing issues without

Reply:
I'm not sure. This is the first run I'm trying. I'll try with and without patches.